aggregates/src/androidMain/kotlin/org/dhis2/mobile/aggregates/data/DataSetInstanceRepositoryImpl.kt

```diff
@@ -897,4 +899,22 @@ internal class DataSetInstanceRepositoryImpl(
         }
         return dataToReview
     }
+
+    private fun getDataElementInfoLabel(
+        dataElement: DataElement,
+        coc: CategoryOptionCombo?,
+    ): String {
+        val isDefaultCategoryCombo = d2.categoryModule().categoryCombos()
+            .uid(coc?.categoryCombo()?.uid())
+            .blockingGet()
+            ?.isDefault ?: false
+
+        val dataElementLabel = dataElement.run { displayFormName() ?: displayName() ?: uid() }
```

Comment:
This is just to avoid the nullable warning. The formName should never be null though.